src/Request.php

```diff
@@ -53,12 +53,11 @@ public function connect()
                 $this->getConnectOption('login_method', 'AMQPLAIN'),
                 $this->getConnectOption('login_response', null),
                 $this->getConnectOption('locale', 3),
+                $this->getConnectOption('channel_rpc_timeout', 0.0),
```

Comment:
I think this should be
```suggestion
                $this->getConnectOption('connection_timeout', 3.0),
```